sig-docs/charter.md

```diff
@@ -0,0 +1,97 @@
+# SIG Docs Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and adheres to the Roles and Organization Management specified in [sig-governance].
+
+## Scope
+
+SIG Docs publishes Kubernetes documentation on kubernetes.io. Kubernetes documentation includes:
+  - Documentation of the core Kubernetes APIs
+  - Core Kubernetes architecture 
+  - CLI tools shipped with the Kubernetes release
+
+ Responsibility for creating feature documentation belongs to the developers and SIGs associated with a feature. This includes documentation for features, concepts, components, opinions, processes, etc.
+
+SIG Docs sets standards for feature documentation, provides clear paths for docs contribution, and [coordinates documentation updates][docs-release] during quarterly releases.
+
+SIG Docs maintains the Kubernetes website's infrastructure and tooling.
+
+SIG Docs creates subprojects as needed to handle specific aspects of Kubernetes documentation.
+
```

Comment:
Suggest added something about content maintenance: SIG Docs is responsible for maintaining existing content and UX on the site.